README.md

```diff
@@ -1,501 +1,69 @@
+### Parsers and Language Support
 
-WDL also lets you define more advanced structures, like the ability to call a task in parallel (referred to as 'scattering').  In the example below, the `wc` task is being called n-times where n is the length of the `Array[String] str_array` variable.  Each element of the `str_array` is used as the value of the `str` parameter in the call to the `wc` task.
+- Basic parsers and their grammar definitions (based on hermes) can be found in the `parsers/` directory for each respective version. Currently there is support for java, python and javascript. We believe these parsers work, however have not validated these claims.
+- [MiniWDL](https://github.com/chanzuckerberg/miniwdl)] - MiniWDL provides python bindings for WDL as well as command line validation. It is light weight and easy to use.
+- [wdl4s](https://github.com/broadinstitute/cromwell) - scala bindings for WDL. This has been folded into the broader cromwell codebase
+- [WOMTool](https://cromwell.readthedocs.io/en/stable/WOMtool/) - a standalone tool for parsing, validating, linting, and generating a graph of a WDL.
+- [wdl-aid](https://github.com/biowdl/wdl-aid) - generate documentation for the inputs of WDL workflows, based on the parameter_meta information defined in the WDL file.	
 
-```wdl
-task wc {
-  String str
-  command {
-    echo "${str}" | wc -c
-  }
-  output {
-    Int count = read_int(stdout()) - 1
-  }
-}
+### IDE Support
 
-workflow wf {
-  Array[String] str_array
-  scatter(s in str_array) {
-    call wc {
-      input: str=s
-    }
-  }
-}
-```
+- Visual Studio Code: [WDL Syntax Highlighter](https://marketplace.visualstudio.com/items?itemName=broadinstitute.wdl)
+- JetBrains IDE's: [Winstanly](https://plugins.jetbrains.com/plugin/8154-winstanley-wdl)
+- Atom: [Language-WDL](https://atom.io/packages/language-wdl)
+- Vim: [vim-wdl](https://github.com/broadinstitute/vim-wdl)
 
-# Getting Started with WDL
 
-We'll use [Cromwell](https://github.com/broadinstitute/cromwell) and [wdltool](https://github.com/broadinstitute/wdltool) to run these examples but you can use any WDL engine of your choice.
+# Contributing
 
-If you don't already have a reference to the Cromwell JAR file, one can be [downloaded](https://github.com/broadinstitute/cromwell/releases)
+WDL only advances through community contributions. While submitting an issue is a great way to report a bug in the spec, or create disscussion around current or new features, it will ultimately not translate into an actual change in the spec. The best way to make changes is by submitting a PR. For more information on how you can contribute, please see the [Contributing](CONTRIBUTING.md) readme. 
 
-If you don't already have a reference to the wdltool JAR file, one can be [downloaded](https://github.com/broadinstitute/wdltool/releases)
+Additionally, once a PR has been submitted, it will be subjected to our [RFC Process](RFC.md).
```

Comment:
Ah I see what you did here re the RFC stuff. I still think it's worth an explicit link from the governance doc as well